components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,387 @@
+                      data-cy="ccf-org-website"
+                    >
+                      {inputProps => (
+                        <Field
+                          onChange={e => {
+                            setFieldValue('website', e.target.value);
+                          }}
+                          as={StyledInputGroup}
+                          {...inputProps}
+                          prepend="http://"
+                          placeholder={intl.formatMessage(placeholders.website)}
+                        />
+                      )}
+                    </StyledInputField>
+                  </Flex>
+                  <Flex flexDirection="column" width={[320, 350, 376, 476]} mx={[1, 10, 15]}>
+                    <H4>
+                      <FormattedMessage id="organization.coadmins.headline" defaultMessage="Administrators" />
+                    </H4>
+                    <P fontSize="14px" mb={2} lineHeight={2}>
+                      <FormattedMessage
+                        id="coAdminsDescription"
+                        defaultMessage="Organization admins can make changes in the profile and interact with other profiles on behalf of this organization."
                       />
-                    ),
-                )}
-              </div>
-            ))}
-
-            <div className="tos">
-              <label>{intl.formatMessage(this.messages['tos.label'])}</label>
-              <div data-cy="tos">
-                <StyledCheckbox
-                  name="tos"
-                  id="oc-tos-checkbox"
-                  required
-                  onChange={({ target }) => this.handleChange('tos', target.checked)}
-                  label={
-                    <FormattedMessage
-                      id="createcollective.tos.label"
-                      defaultMessage="I agree with the {toslink} of Open Collective."
-                      values={{
-                        toslink: (
-                          <StyledLink href="/tos" openInNewTab>
-                            <FormattedMessage id="tos" defaultMessage="terms of service" />
-                          </StyledLink>
-                        ),
-                      }}
-                    />
-                  }
-                />
-              </div>
-            </div>
+                    </P>
+                    <Container border="1px solid #E6E8EB" borderRadius="8px" p={[2, 3]} height="auto">
+                      <Flex flexDirection="row" alignItems="center" justifyContent="space-around">
+                        <Flex fontSize="10px" mr={2}>
+                          <FormattedMessage id="inviteAdmin" defaultMessage="INVITE CO-ADMIN" />
+                        </Flex>
+                        <StyledHr flex="1" borderStyle="solid" borderColor="black.300" width={[100, 110, 120]} />
+                      </Flex>
+                      <Flex data-cy="org-profile-card">
+                        <OnboardingProfileCard
```

Comment:
Double check the alignment, wrapping, and spacing of the profile cards when you add a lot.
<img width="476" alt="Screenshot 2020-10-07 at 11 05 11" src="https://user-images.githubusercontent.com/37520401/95321493-03afce00-0893-11eb-8ccc-4727f3650202.png">

Reply:
Done